src/Avalonia.Visuals/Media/FormattedText.cs

```diff
@@ -10,128 +10,109 @@ namespace Avalonia.Media
     /// <summary>
     /// Represents a piece of text with formatting.
     /// </summary>
-    public class FormattedText : AvaloniaDisposable
+    public class FormattedText
```

Comment:
Why is it not disposable anymore? At least for Direct2D and Cairo underlying implementations are.